packages/preset-create-react-app/package.json

```diff
@@ -33,12 +33,14 @@
   },
   "devDependencies": {
     "@storybook/node-logger": "^6.3.2",
+    "@storybook/core-common": "^6.3.2",
     "@types/semver": "^7.3.6",
     "typescript": "^4.1.3"
   },
   "peerDependencies": {
     "@babel/core": "*",
     "@storybook/node-logger": "*",
+    "@storybook/core-common": "*",
```

Comment:
does this mean that the user's project has to depend on core-common? @yannbf

Reply:
Yes, although core-common is a part of every app package. Norbert just mentioned this could be a problem with pnp, so I guess we can:
1 - move it to be a dependency
2 - remove it, as I just added it to comply with the types requested in the code comments previously left there

Reply:
yeah i was thinking of pnp etc. i'd vote to remove it